plugins/teststeps/sshcmd/sshcmd.go

```diff
@@ -101,6 +104,18 @@ func (ts *SSHCmd) Run(cancel, pause <-chan struct{}, ch test.TestStepChannels, p
 			return fmt.Errorf("failed to convert port parameter to integer: %v", err)
 		}
 
+		timeoutStr, err := ts.Timeout.Expand(target)
+		if err != nil {
+			return fmt.Errorf("cannot expand timeout parameter: %v", err)
+		}
+
+		timeout, err := time.ParseDuration(timeoutStr)
+		if err != nil {
+			return fmt.Errorf("cannot parse timeout paramter: %v", err)
+		}
+
+		timeTimeout := time.Now().Add(timeout)
```

Comment:
Do we really need param expansion for the timeout? If not we could do the parsing in the validate method, so we catch typos at job ingestion time.